### PR TITLE
OBMIN-332 Update swagger parameter

### PR DESCRIPTION
**Pull request by ennnbaev**
Update swagger parameter
Add produces to endpoint to get a necessary response
Change example for PhoneDto

### CODE REVIEW

File: src/main/java/space/obminyashka/items_exchange/controller/AdvertisementController.java [hunk 1]
```diff
@@ -54,85 +55,85 @@ public class AdvertisementController {
...
     private final LocationService locationService;
 
-    @GetMapping(ApiKey.ADV_THUMBNAIL)
+    @GetMapping(value = ApiKey.ADV_THUMBNAIL, produces = MediaType.APPLICATION_JSON_VALUE)
     @Operation(summary = "Find requested quantity of the advertisement as thumbnails and return them as a page result with filters")
     @ApiResponses(value = {
             @ApiResponse(responseCode = "200", description = "OK"),
             @ApiResponse(responseCode = "400", description = "BAD REQUEST"),
             @ApiResponse(responseCode = "404", description = "NOT FOUND")})
     public Page<AdvertisementTitleDto> findPaginatedAsThumbnails(
```
Comment (rpkyrych) [inline src/main/java/space/obminyashka/items_exchange/controller/AdvertisementController.java:64]: - [ ]  TODO: We need to extract those parameters into a new request entity.

I'd be very grateful if you have the will to do that here.

File: src/main/java/space/obminyashka/items_exchange/controller/AdvertisementController.java [hunk 2]
```diff
@@ -54,85 +55,85 @@ public class AdvertisementController {
...
+            @Parameter(name = "size", description = "Number of records per page. Default value: 12")
             @RequestParam(value = "size", required = false, defaultValue = "12") @PositiveOrZero int size) {
         return advertisementService.findAllThumbnails(excludeAdvertisementId, subcategoryId, PageRequest.of(page, size));
     }
 
-    @GetMapping(ApiKey.ADV_THUMBNAIL_RANDOM)
+    @GetMapping(value = ApiKey.ADV_THUMBNAIL_RANDOM, produces = MediaType.APPLICATION_JSON_VALUE)
     @Operation(summary = "Find N random advertisement as thumbnails and return them as a result with filters")
     @ApiResponse(responseCode = "200", description = "OK")
     public List<AdvertisementTitleDto> findRandom12Thumbnails(
```
Comment (rpkyrych) [inline src/main/java/space/obminyashka/items_exchange/controller/AdvertisementController.java:79]: - [ ] TODO: replace all those parameters with a new request entity as well

File: src/main/java/space/obminyashka/items_exchange/controller/AdvertisementController.java [hunk 3]
```diff
@@ -155,7 +156,8 @@ public List<AdvertisementTitleDto> getFirst10BySearchParameters(@Valid @RequestB
             consumes = {
                     MediaType.APPLICATION_JSON_VALUE,
                     MediaType.MULTIPART_FORM_DATA_VALUE,
-                    MediaType.APPLICATION_OCTET_STREAM_VALUE})
+                    MediaType.APPLICATION_OCTET_STREAM_VALUE},
```
Comment (rpkyrych) [inline src/main/java/space/obminyashka/items_exchange/controller/AdvertisementController.java:159]: Did you have a chance to create a new advertisement from Swagger using that type?

File: src/main/java/space/obminyashka/items_exchange/dto/PhoneDto.java [hunk 4]
```diff
@@ -13,7 +13,7 @@
 @Setter
 @EqualsAndHashCode
 public class PhoneDto {
-    @Schema(requiredMode = Schema.RequiredMode.REQUIRED, example = "+38(123)456-78-90, 38 123 456 78 90")
+    @Schema(requiredMode = Schema.RequiredMode.REQUIRED, example = "381234567890")
```
Comment (rpkyrych) [inline src/main/java/space/obminyashka/items_exchange/dto/PhoneDto.java:16]: 👍

**Review by rpkyrych — APPROVED** [review]

**Review by ImAndVlad — APPROVED** [review]
Good job

**Review by IllyaZuew — APPROVED** [review]

**Review by demus-n — APPROVED** [review]

**Review by Sershok — APPROVED** [review]

**Pull request merged**